modules/user/user.service.js

```diff
@@ -136,7 +136,7 @@ class UserService {
       throw new UserInputError(WRONG_CREDENTIALS, { statusCode: 400 });
     }
 
-    const token = generateToken(user._id, user.email);
+    const token = generateToken(user._id, user.email, { EXPIRES_IN: '10h' });
```

Comment:
Why do you need this change as a part of merge request?

Reply:
There were some issues with token expiring too fast, so i just set it for admin to be longer. Removed.